src/components/TimePicker/TimePicker.jsx

```diff
@@ -64,8 +63,8 @@ type Props = {|
 
 class TimePicker extends React.Component<Props> {
   static defaultProps = {
-    minTime: '08:00',
-    maxTime: '20:00',
+    minTime: '00:00',
+    maxTime: '24:00',
```

Comment:
I'm a bit concerned about allowing both "00:00" and "24:00".
It would work with Moment and get the correct representation, but I don't love it...

Reply:
I guess it's the right choice, though... it would still appear as "00:00" when printed, most likely...

Reply:
its ok, the first and the last value are 12